debugging-queries.mdx

```diff
@@ -168,3 +177,128 @@ def sum_transaction_amount(txns: User.transaction[Transaction.amount]) -> User.s
 You can also view these logs using `resolver_replay`, where they will be emitted to your terminal:
 
 ![Logs output example](/img/debugging/logs.png)
+
+## Query Error Categories
+
+In using the debugging techniques above, you may encounter a few different categories of errors
+that have common root causes:
+
+### Request error
+
+Request errors are raised before execution of your resolver code.
+They may occur due to invalid feature names in the
+input or a request that cannot be satisfied by the resolvers you
+have defined.
+
+
+### Field error
+
+Field errors are raised while running a feature resolver
+for a particular field.
```

Comment:
What is meant by for a particular field, here? Is this like what we have sort of moved to calling feature—like a single feature in a feature set? Maybe something like:

"Feature errors are raised when a specific resolver fails"?